main.js

```diff
@@ -46,6 +48,14 @@ const animate = () => {
   drawPaddles();
   drawBall();
   updatePaddles();
+  ball.x += ballSpeedHorizontal;
+  ball.y += ballSpeedVertical;
+  if (ball.x < 10 || ball.x > 949) {
```

Comment:
`949` is acting as a "magic number" here. Its related to the other magic number we set the canvas width too.

This value should get a `const canvasWidth = 960` value near the top. Then the canvas width should be set to that value.

This makes it easier to update the size without having to search through the code.